brave/brave-core
# Moves wallet logic in shared lib

Resolves https://github.com/brave/brave-browser/issues/11681

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- start browser on mater branch
- enable rewards
- connect to Uphold
- open `preference.json` file and save down payment id, seed and uphold wallet
- switch to this branch
- start browser
- make sure that data in `Preference.json` is migrated in `wallets` object and it's the same

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
